aea/test_tools/docker_image.py

```diff
@@ -127,7 +132,7 @@ def launch_image(
     container.start()
     logger.info(f"Setting up image {image.tag}...")
     success = image.wait(max_attempts, timeout)
-    if not success:
+    if not success:  # pragma: no cover
```

Comment:
Please test this branch

Reply:
I will address in #378, as I need to address a failing test on macOS there anyway, to save on CI time